.github/workflows/pr.yaml

```diff
@@ -344,12 +345,18 @@ jobs:
       id: check
       with:
         mode: check
+    - name: Check if all test chunks succeeded
+      run: |
+        NFILES=$(ls artifacts/ | grep "Tool test output" | wc -l)
+        if [[ "${{ needs.setup.outputs.chunk-count }}" != "$NFILES" ]]; then
+          exit 1
+        fi
 
   # deploy the tools to the toolsheds (first TTS for testing)
   deploy:
     name: Deploy
     needs: [setup, lint, flake8, lintr, combine_outputs]
-    if: ${{ (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main' ) && github.repository_owner == 'workflow4metabolomics' }}
+    if: ${{ (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main' ) && github.repository_owner == 'galaxyproject' }}
```

Comment:
I haven't completely mastered this CI but are you sure of that?

Reply:
No this is wrong. Thanks for checking. Will fix it as soon as possible.

Reply:
```suggestion
    if: ${{ (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main' ) && github.repository_owner == 'workflow4metabolomics' }}
```